docs/features/ethernet-lan.md

```diff
@@ -23,15 +23,17 @@ WLED controllers are usually integrated into local network wirelessly (via WiFi)
 Go to "Config" and then to "WiFi Setup". At the bottom of this page select the Ethernet type you use. Then click on "Save &  Connect". Ready!
 ![](../assets/images/content/wled_ethernet_settings.jpg)
 
-## Ethernet controllers and adapters supported by WLED (as of November 2023)
+## Ethernet controllers and adapters supported by WLED (as of v0.15 development, June 2024)
```

Comment:
This will need to reflect released version, not beta or other development version.

Reply:
Right, I assume we'll hold off on this PR until v0.15 release? So adjust this to something like `v0.15, INSERT_RELASE__DATE`?

Reply:
No date, just release.